### ISSUE TITLE
Shuffle and Repeat Status in Miniplayer

### ISSUE THREAD

**Issue by krisdestruction** [radiant-player/radiant-player-mac#474]
The shuffle and repeat buttons do not update their status when they are clicked. Note in the following screenshot how shuffle is enabled but that status is not reflected in the Miniplayer.
![screen shot 2015-12-30 at 8 18 21 pm](https://cloud.githubusercontent.com/assets/1440068/12059904/8b9cae56-af32-11e5-9087-0eb455565635.png)

Version: 1.6.2

**Comment by jacobwgillespie** [comment]
Yeah, we should look into this one - after #402 lands, we will have a better system for detecting the repeat / shuffle state.

**Comment by chrismou** [comment]
Am looking at fixing this as part of #402 along with the other miniplayer issues
